moveit_planners/trapezoidal/README.md

```diff
@@ -0,0 +1,236 @@
+# Overview
```

Comment:
Can we convert some of this README into a moveit_tutorial?

Reply:
Yes, this is planned. @jschleicher I think the current documentation matches the style of tutorial sections for other planners well. Although we can probably skip the part that introduced moveit. ;-)

Reply:
See ros-planning/moveit_tutorials#461.